tools/wpt/browser.py

```diff
@@ -111,27 +92,46 @@ def install(self, dest=None):
         from mozdownload import FactoryScraper
         import mozinstall
 
+        platform = {
+            "Linux": "linux",
+            "Windows": "win",
+            "Darwin": "mac"
+        }.get(uname[0])
+
+        if platform is None:
+            # This was earlier checked in platform_string, which isn't needed anymore since we switched
```

Comment:
I think this comment should explain why the code is as it is rather than explain the historical context (I'm not actually sure a comment is needed here at all).

Reply:
Alright! Added it just to clarify why it was here instead of in platform_string rather than anywhere else. Will remove it